lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.PlantLoop.rb

```diff
@@ -135,114 +173,367 @@ def plant_loop_apply_prm_number_of_chillers(plant_loop, sizing_run_dir = nil)
+        chiller.setReferenceEnteringCondenserFluidTemperature(OpenStudio.convert(95.0, 'F', 'C').get)
+        chiller.setMinimumPartLoadRatio(0.15)
+        chiller.setMaximumPartLoadRatio(1.0)
+        chiller.setOptimumPartLoadRatio(1.0)
+        chiller.setMinimumUnloadingRatio(0.25)
+        chiller.setChillerFlowMode('ConstantFlow')
+        chiller.setReferenceCOP(default_cop)
+
+        condenser_water_loop.get.addDemandBranchForComponent(chiller) if condenser_water_loop.is_initialized
+
       end
+
+      chiller.setName("#{template} #{chiller_cooling_type} #{chiller_compressor_type} Chiller #{i + 1} of #{num_chillers}")
+      chiller.setSizingFactor(per_chiller_sizing_factor)
+      chiller.setReferenceCapacity(per_chiller_cap_w)
+      chiller.setCondenserType(chiller_cooling_type)
+      chiller.additionalProperties.setFeature('compressor_type', chiller_compressor_type)
+
+      # Add inlet pump
+      new_pump = OpenStudio::Model::PumpVariableSpeed.new(plant_loop.model)
+      new_pump.setName("#{chiller.name} Inlet Pump")
+      new_pump.setRatedPumpHead(original_pump.ratedPumpHead / num_chillers)
+      new_pump.setCoefficient1ofthePartLoadPerformanceCurve(original_pump.coefficient1ofthePartLoadPerformanceCurve)
+      new_pump.setCoefficient2ofthePartLoadPerformanceCurve(original_pump.coefficient2ofthePartLoadPerformanceCurve)
+      new_pump.setCoefficient3ofthePartLoadPerformanceCurve(original_pump.coefficient3ofthePartLoadPerformanceCurve)
+      new_pump.setCoefficient4ofthePartLoadPerformanceCurve(original_pump.coefficient4ofthePartLoadPerformanceCurve)
+      chiller_inlet_node = chiller.connectedObject(chiller.supplyInletPort).get.to_Node.get
+      new_pump.addToNode(chiller_inlet_node)
+
     end
 
-    # Ensure there is only 1 chiller to start
-    first_chiller = nil
-    return true if chillers.size.zero?
+    # Remove original pump, dedicated chiller pumps have all been added
+    original_pump.remove
 
-    if chillers.size > 1
-      OpenStudio.logFree(OpenStudio::Error, 'prm.log', "For #{plant_loop.name}, found #{chillers.size} chillers, cannot split up per performance rating method baseline requirements.")
-    else
-      first_chiller = chillers[0]
+    OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.PlantLoop', "For #{plant_loop.name}, there are #{chillers.size} #{chiller_cooling_type} #{chiller_compressor_type} chillers.")
+
+    # Check for a heat exchanger fluid to fluid-- that lets you know if this is a primary loop
+    has_secondary_plant_loop = !plant_loop.demandComponents(OpenStudio::Model::HeatExchangerFluidToFluid.iddObjectType).empty?
+
+    if has_secondary_plant_loop
+      # Add EMS to stage chillers if there's a primary/secondary configuration
+      if num_chillers > 3
+        OpenStudio.logFree(OpenStudio::Error, 'prm.log', "For #{plant_loop.name} has more than 3 chillers. We do not have an EMS strategy for that yet.")
```

Comment:
We probably want a error message that users could understand, I don't think that they would know what the `EMS strategy would be`.

Reply:
Sure that's fair. I'm not quite sure how to communicate that. The issue I'm trying to communicate is that the EMS code we developed to stage chillers was only written to handle up to 3 chillers. The existing logic in the code cannot handle more than 3 chillers. That should only happen in the case of a requirement of > 2400 tons.

Reply:
We can simply say we cannot handle the scenario when there are more than 3 chiller.
In addition, I am not sure if we should stop processing or force this to be 3 chillers and then let user know so that they can modify the data themselves?
@jugonzal07 @lymereJ  thoughts?

Reply:
That sounds good to me. I'd be in favor of stopping the process so users don't end-up with a non-valid model if they don't look at the log messages.

Reply:
Right-- I believe it's better to stop the process. I can update the language on it to be more generic if you'd like, and not reference the EMS.